src/components/Tabs/ScatterPlotTab.tsx

```diff
@@ -76,43 +71,34 @@ export default memo(function ScatterPlotTab(props: ScatterPlotTabProps): ReactEl
   // ^ Memo prevents re-rendering if the props haven't changed.
   const theme = useContext(AppThemeContext);
 
-  const store = useViewerStateStore(
-    useShallow((state) => ({
-      dataset: state.dataset,
-      colorRampRange: state.colorRampRange,
-      colorRamp: state.colorRamp,
-      categoricalPalette: state.categoricalPalette,
-      inRangeLUT: state.inRangeLUT,
-      featureKey: state.featureKey,
-      currentFrame: state.currentFrame,
-      setFrame: state.setFrame,
-      setTrack: state.setTrack,
-      clearTrack: state.clearTrack,
-      track: state.track,
-    }))
-  );
```

Comment:
I ended up changing this accessor pattern based on a discussion in #dev-chat. Individual selectors allow the linter to flag unused properties.